packages/core/src/interpreter.ts

```diff
@@ -310,9 +325,13 @@ export class Interpreter<
         this._complete();
       }
     } catch (err) {
-      this.observers.forEach((observer) => {
-        observer.error?.(err);
-      });
+      for (const observer of this.observers) {
+        try {
+          observer.error?.(err);
+        } catch (err2) {
+          reportUnhandledError(err2);
+        }
```

Comment:
So the logic is that if _any_ observer is missing an error handler (even if other observers have error handlers), rethrow anyway?

Reply:
Yeah, smth like that. In RxJS would get an error for each observer that is missing an error handler but it would still be the same error, it feels somewhat redundant. In RxJS each observer is something that is notified about the error and calling the error listener is delegated to it, so there is no~ loop over existing observers etc so they can't even apply such logic that would count error listeners on those observers etc (that's not a detail that is surfaced to the code that forwards the error to multiple observers). We have that looping centralized~ so I don't see why we wouldn't leverage that.

Note though that the code that you have quoted here isn't exactly about that. This is an extra `try/catch` that ensures that we report errors that might be thrown by error listeners themselves. At the same time, we ensure that each error listener is notified about the original error.